api/src/routes/user.ts

```diff
@@ -515,11 +515,17 @@ export const userGetRoutes: FastifyPluginCallbackTypebox = (
           where: { userId: req.session.user.id }
         });
 
-        const [userToken, user, completedSurveys] = await Promise.all([
-          userTokenP,
-          userP,
-          completedSurveysP
-        ]);
+        const msUsernameP = fastify.prisma.msUsername.findFirst({
+          where: { userId: req.session.user.id }
```

Comment:
After Oliver's PR, I believe this is what we now want:

```suggestion
          where: { userId: req.user?.id }
```